python/py-gitdb/Portfile

```diff
@@ -13,11 +13,11 @@ license             BSD
 description         GitDB is a pure-Python git object database
 long_description    ${description}
 
-python.versions     27 36 37
+python.versions     27 36 37 38
```

Comment:
additional issue here is that this newest version does not support PY27 anymore, so you'll need to make provision to install an older version for that subport (please check that for the other PRs you submitted as well)

Reply:
other alternative is to actually remove PY27 but then you'll have to make sure to delete all the PY27 subports in the dependency tree as well

Reply:
the number of ports that would need removing is not so large

https://ports.macports.org/port/py27-gitdb/summary

doing that though would require the approval of the maintainers of them, though.

Reply:
these ports are all `nomaintainer`, so that's not an issue

Reply:
`# port list rdepends:py27-gitpython rdepends:py27-gitdb
py-git-sweep                   @0.1.1          python/py-git-sweep
py27-git-sweep                 @0.1.1          python/py-git-sweep
py27-gitpython                 @2.1.11         python/py-gitpython
`

I think the forward thing to do is remove py27 from py-git-sweep.

Reply:
py-git-sweep is dead since 2012, and the author's project homepage is dead, and PRs, including Python 3x porting efforts have been ignored since 2016.

https://github.com/arc90/git-sweep/pull/46